test/util/upgrade.js

```diff
@@ -1033,18 +1049,18 @@ async function getMetaTxPrivateContractState(protocolDiamondAddress) {
         */
 
   // starting slot
-  const metaTxStorageSlot = keccak256(ethers.utils.toUtf8Bytes("boson.protocol.metaTransactions"));
-  const metaTxStorageSlotNumber = ethers.BigNumber.from(metaTxStorageSlot);
+  const metaTxStorageSlot = keccak256(toUtf8Bytes("boson.protocol.metaTransactions"));
+  const metaTxStorageSlotNumber = BigInt(metaTxStorageSlot);
 
   // current sender address + isMetaTransaction (they are packed since they are shorter than one slot)
   // should be always be 0x
-  const inTransactionInfo = await getStorageAt(protocolDiamondAddress, metaTxStorageSlotNumber.add("0"));
+  const inTransactionInfo = await getStorageAt(protocolDiamondAddress, metaTxStorageSlotNumber + "0");
```

Comment:
This concatenates strings, does not add up values. There are multiple occasions below, won't point out each of them.

Reply:
I don't see this change applied.
And just to be exact, this needs to be changed everywhere we have `xxxSlotNumber + "n"` (around 30 cases)

Reply:
done now